pkg/ansible/watches/watches_test.go

```diff
@@ -124,9 +124,6 @@ func TestLoad(t *testing.T) {
 		maxWorkers       int
 		ansibleVerbosity int
 		expected         []Watch
-		setenvvar        bool
-		envvar           string
-		envvarValue      int
```

Comment:
I'm a little surprised to see these just removed, I thought maligned was only about sorting fields.